core/core-awt/src/main/java/org/icepdf/core/pobjects/Page.java

```diff
@@ -305,9 +305,9 @@ private void initPageAnnotations() throws InterruptedException {
             List v = (List) annots;
             annotations = new ArrayList<>(v.size() + 1);
             // add annotations
-            Object annotObj;
-            org.icepdf.core.pobjects.annotations.Annotation a = null;
             for (Object aV : v) {
+                Object annotObj;
```

Comment:
I'm kind of old fashion here, in that I don't like recreating object references when we don't need to.  Not a big deal here but it used to make a small performance gain in blocks that are called a lot more.  Maybe instead there could be an 'else' that takes care of the bug you found but also keeps my old antiquated optimization in place. 
```
if (annotObj instanceof Annotation) {
    a = (Annotation) annotObj;
 }
...
else {
   a = null;
}   
```